bigtop-packages/src/common/logstash/patch0-logstash.diff

```diff
@@ -0,0 +1,2393 @@
+diff -ruaN logstash-5.4.1/bin/bundle linaro-logstash-5.4.1/bin/bundle
```

Comment:
Where does this patch come from and why this is necessary instead of pulling an official released version?

Reply:
Thanks for comment, evans.
1. Pick the logstash version: 5.4.1. It corresponds to the elasticsearch version in Bigtop.
2.  `patch0-logstash.diff` is to support Arm64.
The Jruby-1.9 in logstash-5.4.1 doesn't support Arm64 due to the fact that Jffi 1.2.12 which Jruby 1.9 depends on has no native AArch64 libraries. Also see:  [jffi](https://github.com/jnr/jffi/pull/32), [jruby1](https://github.com/jruby/jruby/pull/4548), [jruby2](https://github.com/jruby/jruby/pull/4567).
So `patch0-logstash.diff` upgraded jruby to 9K and also fix dependency issues for Arm64.
The modification for Arm64 support was archived in [linaro repo](https://git.linaro.org/leg/bigdata/logstash.git) and `patch0-logstash.diff` comes from it.

Reply:
Usually for apache project we just name it like patch0-HBASE-16864.diff, which we can easily find the source.
For this one... how about we name it something like patch0-logstash-for-arm64.diff so that it's more clear. Also describe the source of the repo in Bigtop JIRA helps people to locate the change.

Reply:
Yes, it maskes sense.
Updated, thanks!

Reply:
Sorry I forgot this one. @guyuqi is this ready to merge?

Reply:
@evans-ye  Yes please, thanks a lot!